### PR TITLE
Remove tox cache from previous workspace

### PR DESCRIPTION
**Pull request by vikkyrk**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---
Jenkins doesn't cleanup the previous workspace and since .tox is in the gitignore, explicitly deleting it.

### CODE REVIEW

**Comment by vikkyrk** [comment]
R: @robertwb @jasonkuster @aaltay

**Comment by robertwb** [comment]
Are we losing anything by not preserving this cache?

File: sdks/python/run_postcommit.sh [hunk 1]
```diff
@@ -31,6 +31,9 @@ set -v
 # pip install --user installation location.
 LOCAL_PATH=$HOME/.local/bin/
 
+#Remove any tox cache from previous workspace
```
Comment (aaltay) [inline sdks/python/run_postcommit.sh]: Add a space after `#`

**Comment by aaltay** [comment]
LGTM, thank you.

**Comment by jasonkuster** [comment]
No -- it ought to be wiped out every time but Jenkins doesn't have the appropriate plugin installed so this is the quickest solution currently.

**Comment by jasonkuster** [comment]
LGTM

File: sdks/python/run_postcommit.sh (hunk 1 above)
Comment (vikkyrk) [inline sdks/python/run_postcommit.sh]: Fixed

**Comment by vikkyrk** [comment]
This PR makes the behavior same across pre and post commit. I don't understand how the tox caching actually works but unless post submit execution times significantly increase, this is a clean option.

**Comment by jasonkuster** [comment]
via a completely unscientific sampling,

3 builds after cache deleted: 6:30, 5:43, 6:03

3 successful builds before cache deleted: 7:57, 6:40, 6:56

Doesn't look like it's a lot if it did change at all.

**Comment by robertwb** [comment]
@vikkyrk Please close this manually, as apache doesn't give us write permissions to the repo.

**Pull request closed without merging**